pkg/api/latest/dynakube/extension/spec.go

```diff
@@ -1,15 +1,26 @@
-package dynakube
+package extension
 
 import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/shared/image"
 	corev1 "k8s.io/api/core/v1"
 )
 
-// +kubebuilder:validation:Optional
-type ExtensionsSpec struct {
+type Extensions struct {
```

Comment:
to be consisten, here we call it Extensions, so I prefer to do same everywhere, already commented.